src/community/jdbcconfig/pom.xml

```diff
@@ -43,7 +43,11 @@
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>commons-dbcp</groupId>
+      <artifactId>commons-dbcp</artifactId>
+      <version>1.4</version>
```

Comment:
@NielsCharlier for this dependency, please use dependencyManagement in the top-level pom. This is the version used in GeoTools so should be consistent. Note that community/jms-cluster/activemqBroker/pom.xml is currently using 1.3 and should also use dependencyManagement.